### PR TITLE
Adding transform for smoother FE integration

### PR DESCRIPTION
**Pull request by kjduensing**
Added a transform so the frontend can be integrated more smoothly

### CODE REVIEW

**Comment by ryan-mcneil** [comment]
@kjduensing Are we getting test coverage with those new methods that were added? It would probably be beneficial to see that at least one transform is working as expected. Thanks!

File: lib/lighthouse/letters_generator/service.rb [hunk 1]
```diff
@@ -32,6 +32,41 @@ class Service < Common::Client::Base
...
+          else
+            acc.merge({ k => v })
+          end
+        end
+
+        # Don't return chapter35EligibilityDateTime
+        # It's not used on the frontend, and in fact causes problems
+        transformed_info.merge(
+          { 'monthlyAwardAmount' => info['monthlyAwardAmount']['value'] }
+        ).except('chapter35EligibilityDateTime')
```
Comment (kjduensing) [inline lib/lighthouse/letters_generator/service.rb:67]: This is so cool. Ruby is so cool.

**Review by samcoforma — APPROVED** [review]

**Comment by kjduensing** [comment]
@ryan-mcneil 
> Are we getting test coverage with those new methods that were added?

Good point, I added some tests. Would you mind reviewing/approving when you get the chance? Thanks!

**Review by ryan-mcneil — APPROVED** [review]
LGTM!

**Pull request merged**